lib/values/keys.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class Keys {
+  static const ValueKey scanForDevicesButton = ValueKey('scanForDevicesButton');
```

Comment:
Generic name, please think of something else.

Also consider of changing this class to enum with string values.